### PR TITLE
Sqlite

### LINKED ISSUE #6: Implement SQLite on phone database to store contact info

### LINKED ISSUE #8: Write generateInitials function to make initials for contact card
func generateInitials(fullname: String) -> String {
        return "JA"
        // TODO: Implement this function
        // such that we can extract JA from Julius Alexander
        // Julius Alexander IV --> JA
        // Raymond Jacobson --> RJ
        // Raymond Shu Jacobson --> RJ
        // Raymond --> R
        // Raymond Jacob Jingle Heimer Schmitt --> RS
    }

### PR DESCRIPTION
**Pull request by raymondjacobson**
closes #6 

Still need to implement methods in database.swift to be used for contact interaction. However this PR does get SQLite working thru Cocoapods, storing contacts, rendering cards from contacts, adding new contacts, etc.

```

    func getContact(id: Int64) -> Contact {

    }

    func toggleAutoShare(name: String) {

    }

    func updateRequestedCount(name: String) {

    }

```

EDIT: Consequently closes #8  as well :-)

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PODS:
```
Comment (juliusiv) [inline Podfile.lock:1]: Should this file be gitignored?

Comment (raymondjacobson) [inline Podfile.lock:1]: Don't think so, 

http://stackoverflow.com/questions/9446644/what-goes-into-your-gitignore-if-youre-using-cocoapods

File: whereuat-ios/ContactView.swift [hunk 2]
```diff
@@ -15,17 +15,27 @@ class ContactView: UIView {
...
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
     func initialize() {
         // Initialize views
-        displayContactView = DisplayContactView(color: ColorWheel.randomColor(), contactName: "Raymond Jacobson")
-        editContactView = EditContactView(contactName: "Raymond Jacobson") // TODO: Make this stored and dynamic!
+        displayContactView = DisplayContactView(color: self.contactColor, contactData: contactData)
+        editContactView = EditContactView(contactData: contactData) // TODO: Make this stored and dynamic!
```
Comment (kangp3) [inline whereuat-ios/ContactView.swift]: Delete the TODO here

File: whereuat-ios/ContactsViewController.swift [hunk 3]
```diff
@@ -16,7 +16,10 @@ class ContactsViewController: UICollectionViewController, CNContactPickerDelegat
 
     private let reuseIdentifier = "ContactCell"
     
-    var contactData = ["PK", "JA", "SW", "RJ", "RM", "SR", "PP", "EE", "JJ"]
+    // Set up Database as Singleton
```
Comment (kangp3) [inline whereuat-ios/ContactsViewController.swift:19]: SINGLETONNNNN!!!!

File: whereuat-ios/Database.swift [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        self.autoShare = autoShare
+        self.requestedCount = requestedCount
+    }
+    
+    // This function gets a concatinated version of a contact's first and last names
+    func getName() -> String {
+        return self.firstName + " " + self.lastName
+    }
+    
+    // This function takes a fullname argument, like "Spetelius Ray" and returns "SR"
```
Comment (kangp3) [inline whereuat-ios/Database.swift]: This comment still accurate?

File: whereuat-ios/Database.swift [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    
+    init(firstName: String, lastName: String, phoneNumber: String, autoShare: Bool, requestedCount: Int) {
+        self.firstName = firstName
+        self.lastName = lastName
+        self.phoneNumber = phoneNumber
+        self.autoShare = autoShare
+        self.requestedCount = requestedCount
+    }
+    
+    // This function gets a concatinated version of a contact's first and last names
```
Comment (kangp3) [inline whereuat-ios/Database.swift]: sp: concatenated

File: whereuat-ios/ContactView.swift (hunk 2 above)
Comment (raymondjacobson) [inline whereuat-ios/ContactView.swift]: ack

File: whereuat-ios/Database.swift (hunk 4 above)
Comment (raymondjacobson) [inline whereuat-ios/Database.swift]: ack

File: whereuat-ios/Database.swift (hunk 5 above)
Comment (raymondjacobson) [inline whereuat-ios/Database.swift]: ack

**Pull request merged**